deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/layers/recurrent/LSTMHelpers.java

```diff
@@ -249,6 +251,15 @@ static public FwdPassReturn activateHelper(final Layer layer, final NeuralNetCon
                 currHiddenUnitActivations.muliColumnVector(timeStepMaskColumn);
                 currentMemoryCellState.muliColumnVector(timeStepMaskColumn);
             }
+           // boolean layerNormalization = true;
+            if (useLayerNormalization) {
+                Number stdev = currHiddenUnitActivations.stdNumber();
```

Comment:
Don't need masks here thankfully (it's applied per time step) - that makes the implementation simpler.
@agibsonccc is totally correct about it being activations per time step. That means: currHiddenUnitActivations.std(1), .mean(1) and muliColumnVector/diviColumnVector.